atomic_defi_design/Dex/Constants/General.qml

```diff
@@ -356,10 +356,10 @@ QtObject {
         return trail_zeros ? full_double : full_double.replace(/\.?0+$/,"")
     }
 
-    function formatCrypto(received, amount, ticker, fiat_amount, fiat) {
```

Comment:
I think for the moment we should keep the old function for backwards compatibility and use a new one for notifications because `formatCrypto` function is used in many places inside the fontend and we need to check it didn't break anything